OrchardCore.Commerce.Tests/Fakes/TestMoneyService.cs

```diff
@@ -3,17 +3,18 @@
 using OrchardCore.Commerce.Services;
 using OrchardCore.Commerce.Settings;
 
-namespace OrchardCore.Commerce.Tests.Fakes
+namespace OrchardCore.Commerce.Tests.Fakes;
+
+public class TestMoneyService : MoneyService
 {
-    public class TestMoneyService : MoneyService
-    {
-        public TestMoneyService()
-            : base(new ICurrencyProvider[]
+    public TestMoneyService()
+        : base(
+            new ICurrencyProvider[]
             {
                 new CurrencyProvider(),
-                new AnkhMorporkCurrencyProvider()
+                new AnkhMorporkCurrencyProvider(),
             },
             new TestOptions<CommerceSettings>(new CommerceSettings { DefaultCurrency = "EUR" }),
-            new NullCurrencySelector()) { }
-    }
+            new NullCurrencySelector())
+    { }
```

Comment:
Remember how we discussed using;
```suggestion
    {
    }
```
?

Does Rider not format it like this for you?

Reply:
It was already like that, and neither Rider nor VS triggers code formatting from quick actions. (I'm pretty sure the newline here was added with a solution-wide cleanup action.)